tests/utils/osm_to_gherkin.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
```

Comment:
Can you divide this file to several functions to make it more readable?

Reply:
Done.